package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -766,14 +767,18 @@ def _getOneDeltaPoint(self,universe, repInd, i ,t0, dt):
             valHH += self.lg2(np.dot(unitHHVector0,unitHHVectorp))
             valdip +=  self.lg2(np.dot(unitdipVector0,unitdipVectorp))
             n += 1
-        valOH = valOH/n
-        valHH = valHH/n
-        valdip = valdip/n
-        return (valOH,valHH,valdip)
+
+        if n == 0:
+            return (0,0,0)
```

Comment:
We decided to go with 79 characters, see [Style Guide: Coding Style](https://github.com/MDAnalysis/mdanalysis/wiki/Style-Guide#coding-style) – there was quite a bit of discussion preceding this decision. Generally speaking, I find that most code still looks fine when broken sensibly across lines.

A lot of old code still does not conform to PEP 8 but we are slowly working to making this more uniform so at least newly added/edited code should conform.

In the end, a unified appearance makes it easier for other people to understand the code and it can make merging easier, too.